### PR TITLE
fix display field not shown in value relation

### PR DESCRIPTION
**Pull request by 3nids**
backport of #979

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-0aafd01503b3733dad1f11bf3ea49078d7d3a574-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-0aafd01503b3733dad1f11bf3ea49078d7d3a574-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-0aafd01503b3733dad1f11bf3ea49078d7d3a574-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-0aafd01503b3733dad1f11bf3ea49078d7d3a574-x86_64.apk)

File: src/core/featurechecklistmodel.cpp [hunk 1]
```diff
@@ -33,6 +33,7 @@ QVariant FeatureCheckListModel::data( const QModelIndex &index, int role ) const
     return mCheckedEntries.contains( FeatureListModel::data( index, FeatureListModel::KeyFieldRole ).toString() );
   }
   else
+    qDebug() << index.row() << role << FeatureListModel::data( index, role );
```
Comment (m-kuhn) [inline src/core/featurechecklistmodel.cpp:36]: Same question again ;)

Comment (3nids) [inline src/core/featurechecklistmodel.cpp:36]:
```suggestion
```

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-ad93516fe16170f76517df5e17eb57a3f508f45a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-ad93516fe16170f76517df5e17eb57a3f508f45a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-ad93516fe16170f76517df5e17eb57a3f508f45a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-980-ad93516fe16170f76517df5e17eb57a3f508f45a-x86_64.apk)

**Pull request merged**